Clients/GUI/Scripts/SignUper.cs

```diff
@@ -0,0 +1,28 @@
+﻿using GUI.GUIModels;
+using System.Net;
+using DTO;
+using System.IO;
+using System.Text.Json;
+
+namespace GUI.Scripts
+{
+    public class SignUper
```

Comment:
Strange name. What does it mean? this -er suffix

Reply:
drive - driver, buy -buyer, SignUp - SignUper

Reply:
According to this logic, mb SignUpper then?

Reply:
yes, you are right